extensions/google-cloud-functions-http/runtime/src/main/java/io/quarkus/gcp/functions/http/QuarkusHttpFunction.java

```diff
@@ -100,6 +106,8 @@ private void nettyDispatch(HttpRequest request, HttpResponse response)
             nettyRequest.headers().add(header.getKey(), header.getValue());
         }
 
+        nettyRequest.headers().set(USER_AGENT, USER_AGENT_VALUE);
```

Comment:
@mfpc It also exposes the implementation detail, not every customer will want to tell with USER_AGENT that Quarkus is used